src/modules/land_detector/LandDetector.cpp

```diff
@@ -71,15 +63,16 @@ LandDetector::~LandDetector()
 
 void LandDetector::start()
 {
-	_check_params(true);
+	_force_param_update = true;
```

Comment:
Why do you use a class attribute here? It adds another state, whereas passing an argument would be simpler.

Reply:
Hi @bkueng , I changed it to meet your suggestion and flight testing is good: https://review.px4.io/plot_app?log=a7b84e5b-14dc-4324-a324-986aac6873dc

Initially I had overloaded the method to allow inheritance from the base class while preserving the original behavior.  This was changed after feedback above to use a class variable.  As it stands now, the MulticopterLandDetector class doesn't use the argument, but I don't see any issues with that.

Thanks for the feedback!